src/app/app.module.ts

```diff
@@ -25,6 +25,7 @@ import { IndexComponent } from './index/index';
 import { LoginComponent } from './login/login';
 import { PenaltiesComponent, GpsComponent } from './penalties';
 import { PenaltyOverviewComponent } from './penalties/penalty-overview';
+import { PenaltiesPagination } from './penalties/penalties-pagination';
```

Comment:
Would have gone with some general purpose pagination component, maybe `cpr-pagination`, or plain `./pagination/index.ts`, but fine for now :)